lib/Runtime/ByteCode/ByteCodeGenerator.cpp

```diff
@@ -1267,6 +1267,11 @@ static Js::FunctionInfo::Attributes GetFunctionInfoAttributes(ParseNodeFnc * pno
     if (pnodeFnc->IsMethod())
     {
         attributes = (Js::FunctionInfo::Attributes)(attributes | Js::FunctionInfo::Attributes::Method);
+
+        if (!pnodeFnc->IsClassConstructor())
```

Comment:
dropping a small comment why this is not needed for class would be good. (referencing the spec would be a nice here).